modules/__tests__/matchRoutes-test.js

```diff
@@ -291,10 +291,11 @@ describe('matchRoutes', function () {
   })
 
   describe('an asynchronous JSX route config', function () {
-    let getChildRoutes, getIndexRoute, jsxRoutes
+    let getChildRoutes, getIndexRoute, jsxRoutes, jsxNestedRoutes, state
 
     beforeEach(function () {
-      getChildRoutes = function (location, callback) {
+      getChildRoutes = function (partialNextState, callback) {
```

Comment:
Use `expect.createSpy` here: https://github.com/mjackson/expect#spies

You can then assert on `getChildRoutes.calls[0].arguments[0]`